dlgr/griduniverse/experiment.py

```diff
@@ -1318,11 +1318,24 @@ def game_loop(self):
         self.publish({'type': 'stop'})
         return
 
-    def analyze(self, data):
-        return self.average_score(data)
+    def player_feedback(self, data):
+        df = data.infos.df
+        engagement = int(json.loads(df.questions.list[-1][-1])['engagement'])
```

Comment:
I think you can get the infos table in other formats directly, e.g., `data.infos.list` or `data.infos.json`. See dallinger/data.py.